dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -0,0 +1,129 @@
+            javax.enterprise.inject.spi;resolution:=optional, \
+            javax.imageio;resolution:=optional, \
+            javax.imageio.stream;resolution:=optional, \
+            javax.jms;resolution:=optional, \
+            javax.json;resolution:=optional, \
+            javax.json.stream;resolution:=optional, \
+            javax.management;resolution:=optional, \
+            javax.management.openmbean;resolution:=optional, \
+            javax.naming;resolution:=optional, \
+            javax.persistence;jpa="3.0";consumer="persistenceService";resolution:=optional, \
+            javax.persistence.criteria;jpa="3.0";consumer="persistenceService", \
+            javax.persistence.metamodel;jpa="3.0";consumer="persistenceService", \
+            javax.persistence.spi;jpa="3.0";consumer="persistenceService";resolution:=optional, \
+            javax.resource;resolution:=optional, \
+            javax.resource.cci;resolution:=optional, \
+            javax.rmi;resolution:=optional, \
+            javax.rmi.CORBA;resolution:=optional, \
+            javax.sql;resolution:=optional, \
+            javax.transaction;resolution:=optional, \
+            javax.transaction.xa;resolution:=optional, \
+            javax.validation;resolution:=optional, \
+            javax.validation.groups;resolution:=optional, \
+            javax.validation.metadata;resolution:=optional, \
+            javax.xml.datatype;resolution:=optional, \
+            javax.xml.namespace;resolution:=optional, \
+            javax.xml.parsers;resolution:=optional, \
+            javax.xml.stream;resolution:=optional, \
+            javax.xml.stream.events;resolution:=optional, \
+            javax.xml.transform;resolution:=optional, \
+            javax.xml.transform.dom;resolution:=optional, \
+            javax.xml.transform.sax;resolution:=optional, \
+            javax.xml.transform.stream;resolution:=optional, \
+            javax.xml.validation;resolution:=optional, \
+            javax.xml.xpath;resolution:=optional, \
+            org.eclipse.persistence.jpa, \
+            org.omg.CORBA;resolution:=optional, \
+            org.omg.CORBA.portable;resolution:=optional, \
+            org.omg.CORBA_2_3.portable;resolution:=optional, \
+            org.osgi.framework, \
+            org.w3c.dom;resolution:=optional, \
+            org.xml.sax;resolution:=optional, \
+            org.xml.sax.ext;resolution:=optional, \
+            org.xml.sax.helpers;resolution:=optional
+
+# Use dynamicImport-Package for JAXB APIs, with this, Equinox will have a chance to wire
+# those packages to the JAXB-2.2 API if the target bundle is started, or the one from system
+# bundle will be always used due to JAXB 2.2 API is not resolved
+DynamicImport-Package: \
+ javax.activation,\
+ javax.xml.bind, \
+ javax.xml.bind.annotation, \
+ javax.xml.bind.annotation.adapters, \
+ javax.xml.bind.attachment, \
+ javax.xml.bind.helpers, \
+ javax.xml.bind.util
```

Comment:
this can be removed because we will replace with a hard dependency on jaxb-3.0 once available

Reply:
noted